applications/ipc_radio/README.rst

```diff
@@ -89,24 +90,20 @@ The following files are available:
 * :file:`overlay-bt_hci_ipc.conf` - Configuration file enabling Bluetooth Low Energy over HCI.
 * :file:`overlay-bt_rpc.conf` - Configuration file enabling Bluetooth Low Energy over RPC.
 
-Building and running
-********************
+.. note::
+   When using Sysbuild, the configuration files are added automatically depending on the Sysbuild configuration.
```

Comment:
I'm having trouble understand this sentence, which previously was featured lower in the file.

I was wondering about "When using Sysbuild, the configuration files are added automatically" before (aren't all added automatically when you generate a build configuration?), but now the addition of "depending on the sysbuild configuration" makes me even more confused.

Real-world example of how I read this sentence: "When you use the light switch, the bulb is turned on automatically depending on the light switch configuration".

What does "depending on the sysbuild configuration" add here? Sounds a lot like a tautology...

Reply:
What configuration files are added automatically? Sysbuild ones? Application ones? Are these different than the "sysbuild configuration" later in the sentence?

Reply:
The application (as in specifically and only application) can be built with ready configurations overlay files overlay-802154 etc.
If the application is built as a single image for network core the built process requires to use those overlay files in command line (or via VSC extension) -DEXTRA_CONF_FILE="overlay-abc"

However, when you build some bluetooth sample for instance peripheral_lbs with ipc_radio as network core image, you need only to select *sysbuild* config for SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC and sysbuild Cmake files automatically configure the ipc_radio subimage to include the proper overlay file (in this case overlay-bt_hci_ipc.conf"

Reply:
You can build this ipc_radio application as a single program for netcore. In this case you need to manually specify configuration and/or overlays.

You can also use build bluetooth/ieee802154 application and use sysbuild to automatically configure the ipc_radio for you. In that case the overlays are managed by the sysbuild Cmake and Kconfig files.